RELEASE_NOTES.md

```diff
@@ -1,9 +1,5 @@
 # Frequenz Python SDK Release Notes
 
-## Summary
-
-<!-- Here goes a general summary of what this release is about -->
-
 ## Upgrading
 
 - `frequenz.sdk.config.load_config()` doesn't accept classes decorated with `marshmallow_dataclass.dataclass` anymore. You should use the built-in `dataclasses.dataclass` directly instead, no other changes should be needed, the metadata in the `dataclass` fields will still be used.
```

Comment:
Wasn't this stuff just removed by another PR?

Reply:
https://github.com/frequenz-floss/frequenz-sdk-python/pull/1106

Reply:
Rebased... sorry